Source/missiles.cpp

```diff
@@ -3130,7 +3118,7 @@ void MI_Firebolt(int i)
 				AddMissile(missile.position.tile, { i, 0 }, missile._mimfnum, MIS_MISEXP3, missile._micaster, missile._misource, 0, 0);
 				break;
 			case MIS_BONESPIRIT:
-				SetMissDir(missile, DIR_OMNI);
+				SetMissDir(missile, 8);

```

Comment:
This is not a direction but the explode frame of the missile, only 6/59 of the missiles abusing the directions in this way :tada: the rest spawn an explosion missile on impact.